code/modules/projectiles/ammo_datums.dm

```diff
@@ -1931,6 +1931,41 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
 /datum/ammo/xeno/boiler_gas/corrosive/set_smoke()
 	smoke_system = new /datum/effect_system/smoke_spread/xeno/acid()
 
+/datum/ammo/xeno/larvae_jelly
+	name = "blob of larvae jelly"
+	icon_state = "larvajelly"
+	ping = "ping_x"
+	flags_ammo_behavior = AMMO_XENO|AMMO_SKIPS_ALIENS|AMMO_EXPLOSIVE
+	var/danger_message = "<span class='danger'>A blob of larvae jelly lands with a splat!</span>"
+	armor_type = "bio"
```

Comment:
`ping` and `armor_type` are the same as on `/datum/ammo/xeno`, so they can be removed from here.